src/main/java/frc/robot/subsystems/turretSubsystem/TurretSubsystem.java

```diff
@@ -189,37 +203,38 @@ public TurretSubsystem(RobotStateManager robotStateManager, VisionSubsystem visi
           pitchRoot.append(new MechanismLigament2d("Pitch", 2, 0, 5, new Color8Bit(Color.kBlue)));
       turretTab.add("Pitch", pitchMech);
     }
-
-    currentCommand = new DebugEntry<String>("none", "Turret Command", this);
   }
 
   public void stopTurret() {
-
     turretMotor.stopMotor();
+    pitchMotor.stopMotor();
   }
 
   public double calculateTurretPosition() {
     if (!Constants.enabledSubsystems.turretRotationEnabled) return 0;
     double encoderPosition =
-        lowGearCANcoder.getPosition().getValueAsDouble()
+        (lowGearCANcoder.getPosition().getValueAsDouble() - 0.5)
```

Comment:
why the -0.5?

Reply:
the can coders zero is offset so that the encoder division is off by half a rotation. Making it minor error in either direction won't cause it to be majorly off